### PR TITLE
fix #85631: Range select -> Notes button in Inspector includes grace …

### PR DESCRIPTION
**Pull request by Fyrult**
…notes in list selection, disabling Inspector

The "Notes" button now excludes grace notes. I also added a "Grace Notes" button in the middle for grace notes.

### CODE REVIEW

File: mscore/inspector/inspectorGroupElement.cpp [hunk 1]
```diff
@@ -127,8 +134,36 @@ void InspectorGroupElement::notesClicked()
...
+//---------------------------------------------------------
+
+void InspectorGroupElement::graceNotesClicked()
+      {
+      Score* score = inspector->el().front()->score();
+      QList<Element*> el = score->selection().elements();
+      QList<Element*> nel;
+      score->deselectAll();
+      for (Element* e : el) {
+            if (e->type() == Element::Type::NOTE) {
```
Comment (Jojo-Schmitz) [inline mscore/inspector/inspectorGroupElement.cpp]: New style: `if (e->isNote()) {`. Same a few lines up.

**Comment by Fyrult** [comment]
Fixed styling.

**Comment by lasconic** [comment]
See https://musescore.org/fr/node/85631#comment-522541

File: mscore/inspector/inspectorGroupElement.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [inline mscore/inspector/inspectorGroupElement.cpp]: Bites is at backporting to 2.1 though

**Pull request merged**